### ISSUE TITLE
unable to display DLL module name but instead of ordinal#

### ISSUE THREAD

**Issue by wkingnet** [x64dbg/x64dbg#3117]
### Operating System

windows 10 64bit 19045.3086

### x64dbg Version

Jun 15 2023

### Describe the issue

I updated x64dbg to Jun 15 2023 today,  and found new version unable to display DLL module name but instead of ordinal#.

like this:
```
00AED913 | FF75 08           | push dword ptr ss:[ebp+0x8]                  |
00AED916 | 66:C785 FCFEFFFF  | mov word ptr ss:[ebp-0x104],0x211            |
00AED91F | FF15 E4BAEE00     | call dword ptr ds:[<Ordinal#9>]              |
00AED925 | FF75 FC           | push dword ptr ss:[ebp-0x4]                  |
00AED928 | 66:8985 FEFEFFFF  | mov word ptr ss:[ebp-0x102],ax               |
00AED92F | FF15 E4BAEE00     | call dword ptr ds:[<Ordinal#9>]              |
```

But in fact Ordinal#9 is ws2_32.htons and old version x64dbg should display `call dword ptr ds:[ws2_32.htons]`
![1](https://github.com/x64dbg/x64dbg/assets/57652989/7a4ea8e1-20a9-48c1-bedd-31fe868e26ca)

When I used the old version before upgrading, the display of htons was normal. So is there any way to restore the previous display?

### Steps to reproduce

1. 
2. 
3.

**Comment by mrexodia** [comment]
Yeah this should definitely work... Most likely this was an unintended side effect from the bugfixes related to symbol enumeration. Will take a look when I find the time.

**Comment by mrexodia** [comment]
I cannot reproduce this issue:

![image](https://github.com/x64dbg/x64dbg/assets/2458265/ade8edd8-40a0-486a-bb38-b60ac3d715c3)

**Comment by PolarBears** [comment]
The same problem happened to me, maybe this problem occurs in WS2_32.dll and the system language is chinese?

![image](https://github.com/x64dbg/x64dbg/assets/15968804/ee06b968-80ac-4e72-9468-c50786c7aff9)
![image](https://github.com/x64dbg/x64dbg/assets/15968804/8a735544-c423-43d6-8d23-f47927182f04)
